auth0-rules/add-accountLinkId-to-token.js

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable */
+
+/*
+
+  Function name should be removed for deployment
+
+  uuid is global inside Auth0 Rules
+
+  https://auth0.com/docs/appliance/modules
+
+*/
+
+function addAccountLinkId(user, context, callback) {
+  const namespace = 'https://www.freecodecamp.org/accountLinkId';
```

Comment:
This is just a URL and need not have any actual endpoint?

Reply:
Yes, purely a namespace to avoid collisions. No requests are made to it.

Reply:
In that case I would prefer making this to something like `https://auth-ns.freeCodeCamp.org`